### PR TITLE
Fix http client timeout

### PR DESCRIPTION
**Pull request by tonyredondo**

### CODE REVIEW

File: agent/recorder.go [hunk 1]
```diff
@@ -168,7 +169,7 @@ func (r *SpanRecorder) sendSpans() (error, bool) {
 
 		var testSpans int64
 		for _, span := range spans {
-			if isTestSpan(span) {
+			if isTestSpan(span["tags"].(opentracing.Tags)) {
```
Comment (fermayo) [inline agent/recorder.go:172]: Unrelated change?

Comment (tonyredondo) [reply agent/recorder.go:172]: yes but important one, previously was not counting the test not being sent, and the output message was the "success" one, and not the "error/check the logs" one. I found it while testing the timeout fix.

Comment (fermayo) [reply agent/recorder.go:172]: That's a different PR then

Comment (tonyredondo) [reply agent/recorder.go]: reverted.

File: agent/agent.go [hunk 2]
```diff
@@ -64,7 +64,7 @@ type (
 )
 
 var (
-	version = "0.4.1-pre.1"
+	version = "0.4.1"
```
Comment (fermayo) [inline agent/agent.go:67]: Unrelated change ;)

Comment (tonyredondo) [reply agent/agent.go:67]: -.-
